packages/ai/pyproject.toml

```diff
@@ -21,10 +25,11 @@ authors = [
 repository = "https://github.com/ajtorres9/marvin/tree/master/packages/ai"
 
 [tool.poetry.dependencies]
-python = "^3.7.2"
+python = "^3.7.0"
```

Comment:
This was lowered to `3.7.0` due to the software constraints applied by [Azure Pipelines](https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md)